e2e/ctl_v3_alarm_test.go

```diff
@@ -15,44 +15,62 @@
 package e2e
 
 import (
+	"os"
 	"strings"
 	"testing"
+	"time"
 )
 
-func TestCtlV3Alarm(t *testing.T) { testCtl(t, alarmTest, withQuota(64*1024)) }
+func TestCtlV3Alarm(t *testing.T) {
+	// The boltdb minimum working set is six pages.
+	testCtl(t, alarmTest, withQuota(int64(48*os.Getpagesize())))
```

Comment:
why 48? I think 12 would be better since that gives a tighter bound on the max quota and the quota can be exceeded with a single Put even with 64k pages.